src/tools/kata-ctl/src/ops/check_ops.rs

```diff
@@ -129,3 +133,9 @@ pub fn handle_version() -> Result<()> {
     println!("{} version {:?} (type: rust)", NAME, version);
     Ok(())
 }
+
+pub fn handle_timeout(_args: TimeoutArguments) -> Result<()> {
+    unsafe { KATA_TIMEOUT = _args.timeout };
```

Comment:
There is a bit more overhead in doing so, but I'd recommend using the [`lazy_static`](https://docs.rs/lazy_static/latest/lazy_static) crate instead of this as it avoids the `unsafe` block and is the standard way to manage globals like this. Also, you need to rename `_args` to `args` as you are actually using this parameter (the underscore is used to mean, " I'm not using this thing").

Something like:

```rust
  lazy_static! {
      pub static ref KATA_TIMEOUT_SECS: u64 = 1;
  }

// ...

// Access the global
let timeout_ref = KATA_TIMEOUT.clone();
let mut timeout = timeout_ref;

// And modify it
timeout = 99;
```

Reply:
Thanks your suggestions. @jodh-intel. I am new to rust. To avoid using unsafe, I use mutex to access the shared variable. Using global variable is the way I can think of. Is there a way that doesn't introduce additional overhead？Looking for your advice